website/cue/reference/components/sinks/splunk_hec.cue

```diff
@@ -0,0 +1,47 @@
+package metadata
+
+components: sinks: _splunk_hec: {
+	configuration: {
+		acknowledgements: {
+			common:      false
+			description: "The configuration for Splunk HEC indexer acknowledgement client behavior."
+			required:    false
+			type: object: {
+				options: {
+					query_interval: {
+						common:      false
+						description: "The amount of time to wait in between queries to the Splunk HEC indexer acknowledgement endpoint. Minimum of `1`."
+						required:    false
+						type: uint: {
+							default: 10
+							unit:    "seconds"
+						}
+					}
+					retry_limit: {
+						common:      false
+						description: "The maximum number of times an ack id will be queried for its status. Minimum of `1`."
+						required:    false
+						type: uint: {
+							default: 30
+							unit:    null
+						}
+					}
+				}
+			}
+		}
+	}
+	how_it_works: {
+		indexer_acknowledgements: {
+			title: "Indexer Acknowledgements"
+			body: """
+				For more accurate end-to-end acknowledgements, this sink will automatically integrate with
+				[Splunk HEC indexer acknowledgements](https://docs.splunk.com/Documentation/Splunk/8.2.3/Data/AboutHECIDXAck)
```

Comment:
We use the [`urls.cue`](https://github.com/vectordotdev/vector/blob/ab0744ae8ea1557166257fd1308fac585f751c60/website/cue/reference/urls.cue) file for this. It helps to ensure we have unique links, validate that they work, etc.
```suggestion
				[Splunk HEC indexer acknowledgements](\(urls.splunk_hec_indexer_acknowledgements))
```